src/context/FormContext.tsx

```diff
@@ -153,37 +151,36 @@ export const FormProvider = (props: { children: ReactNode }) => {
 
   const setLoadedEvents = useCallback(
     (events: DevelopmentPlanTimetable[] | null) => {
-      const hasActiveStatusChange = events?.some(
+      setLoadedTimetableEvents(events);
+
+      const currentStatusChangeEvent = events?.find(
         ({ developmentPlanEvent, endDate }) =>
           isStatusChangeEventKey(developmentPlanEvent) && !endDate
       );
 
-      if (hasActiveStatusChange) {
+      if (currentStatusChangeEvent) {
         setStatusHasChanged(true);
```

Comment:
This is called setStatusHasChanged but its more like status is not "progressing normal", right? Just making sure I understand

Reply:
I'm not sure if I've understood your question but `statusHasChanged` directly maps to the answer to this question:
![image](https://github.com/digital-land/local-plans-timetable/assets/29425751/78f0865f-46e4-4aa7-8ed9-d6f2d46ac52f)

Reply:
Okay so if you load a paused timetable then a no on this page would keep it paused or un-pause it?

Reply:
then "yes" will create a new status change event which then gets added in updatedTimetableEvents at the end on ExportPage